src/components/features/cart/pages/home.hbs

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
```

Comment:
every hard-coded text should either be coming from your json or should be stored in a config.js file as constant